Dev-Corner/slides/tool_integration.html

```diff
@@ -45,65 +45,1274 @@
+
+                            <small>[Galaxy Tool XML File](https://docs.galaxyproject.org/en/latest/dev/schema.html)</small>
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Wrapper
+
+                            <img src="../images/wrapper_big_picture_2.png" height=400/>
+
+                            <small>[Galaxy Tool XML File](https://docs.galaxyproject.org/en/latest/dev/schema.html)</small>
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Wrapper
+
+                            <img src="../images/wrapper_big_picture_3.png" height=400/>
+
+                            <small>[Galaxy Tool XML File](https://docs.galaxyproject.org/en/latest/dev/schema.html)</small>
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### `command`
+                            #### How to invoke the tool?
+
+                            ```xml
+                            <command><![CDATA[
+                                graphlan.py
+                                    --format $format
+                                    #if $dpi
+                                        --dpi $dpi
+                                    #end if
+                                    --size $size
+                                    "$input_tree"
+                            ]]></command>
+                            ```
+
+                            <small>If the script is provided near the wrapper xml</small>
+                            ```xml
+                            <command><![CDATA[
+                                python $__tool_directory__/graphlan.py
+                                    --format $format
+                                    #if $dpi
+                                        --dpi $dpi
+                                    #end if
+                                    --size $size
+                                    "$input_tree"
+                            ]]></command>
+                            ```
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### `command`
+                            #### What options to pass?
+
+                            <img src="../images/graphlan_screenshot.png" alt="input"/>
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### `inputs` > `param` -> `command`
```

Comment:
It is not a hierarchie but a sign for exchange: how `param` passes parameters to `command`